dali/python/nvidia/dali/fn.py

```diff
@@ -79,7 +79,7 @@ def fn_wrapper(*inputs, **kwargs):
         from nvidia.dali._debug_mode import _PipelineDebug
         current_pipeline = _PipelineDebug.current()
         if getattr(current_pipeline, '_debug_on', False):
-            return current_pipeline._wrap_op_call(op_wrapper, inputs, kwargs)
+            return current_pipeline._wrap_op_call(op_class, *inputs, **kwargs)
```

Comment:
```suggestion
            return current_pipeline._wrap_op_call(op_class, *inputs, name="_to_snake_case(op_class.__name__)", **kwargs)
```
or it doesn't make any sense?

Reply:
It makes sense, that's the way the names are created [here](https://github.com/ksztenderski/DALI/blob/main/dali/python/nvidia/dali/fn.py#L106), but I think it's even better to just pass the name to the `_wrap_op_class` and have the name generation in one place.